src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.scss

```diff
@@ -1,5 +1,5 @@
 @import 'src/styles/settings';
-@import '../constants';
+@import '../../../constants';
```

Comment:
here, the desirability of the change to `src/content/app/entity-viewer/gene-view/styles/constants` becomes even more pressing